File: Pparty/API/pparty/src/main/java/br/cefet/pparty/model/Mensagem.java
package br.cefet.pparty.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString

public class Mensagem {
    private int idMensagem;
    private String conteudo;
    private int tbChat_IdChat;
    private int udOrigem;

}

File: Pparty/Implemento-Last/src/app/model/mensagem.ts
export class Mensagem {
    idMensagem: number;
    conteudo: string;
    tbChat_IdChat: number;
    udOrigem: number;

    constructor(){
        this.idMensagem = 0;
        this.conteudo = "";
        this.tbChat_IdChat = 0;
        this.udOrigem = 0;
    }
}
